staging/src/k8s.io/kubectl/pkg/cmd/explain/explain_test.go

```diff
@@ -96,60 +104,193 @@ func TestExplainNotExistResource(t *testing.T) {
 	}
 }
 
-func TestExplainNotExistVersion(t *testing.T) {
-	tf := cmdtesting.NewTestFactory()
-	defer tf.Cleanup()
+type explainTestCase struct {
+	Name               string
+	Args               []string
+	Flags              map[string]string
+	Env                map[string]string
```

Comment:
I was going to say that you should probably not use this, but then I realized it's not used.

Reply:
removed